ndb/noxfile.py

```diff
@@ -97,14 +105,6 @@ def lint(session):
     session.run("flake8", "google", "tests")
 
 
-@nox.session(py=DEFAULT_INTERPRETER)
```

Comment:
if you do this, lint will always pass (unless black fails). the ordering insures that lint fails if you push to CI without running 'nox'

Reply:
Ah ok, thanks. Reverted in #6876. Following branches rebased.